.gitlab-ci.yml

```diff
@@ -2806,8 +2786,7 @@ deploy_process_and_sysprobe:
   when: manual
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deploy:$DATADOG_AGENT_BUILDERS
   before_script:
-    - cd $OMNIBUS_PACKAGE_DIR
-    - ls
+    - ls $OMNIBUS_PACKAGE_DIR
```

Comment:
I'm not entirely sure since we don't usually run this job, but I think we do need to `cd $OMNIBUS_PACKAGE_DIR` because that's where the `datadog-agent` deb is. We can move it to the main script.

Reply:
You are right.

Reply:
I've undone the change.